src/components/TechnologiesFeatures/DotnetFeatures/index.tsx

```diff
@@ -0,0 +1,25 @@
+import React from 'react';
+import { OpenFeatureTechnologiesPage } from '../../custom/OpenFeatureTechnologiesPage';
+import CSharpNoFillSvg from '@site/static/img/c-sharp-no-fill.svg';
+
+export class DotnetFeatures
+ extends React.Component {
+  override render() {
+    return <OpenFeatureTechnologiesPage 
+      technology=".NET"
+      svg={CSharpNoFillSvg}
+      sdkRepoLink={{
+        title: 'Github repository',
+        href: 'https://github.com/open-feature/golang-sdk'
+      }}
+      artifact={{
+        title: 'Github repository',
+        href: 'https://github.com/open-feature/golang-sdk',
+        instruction: 'Install-Package OpenFeature',
```

Comment:
I think these days most people would be using the cross-platform dotnet cli instead of the powershell methods.
```suggestion
        instruction: 'dotnet add package OpenFeature',
```